images/mkdocs/requirements-docs.txt

```diff
@@ -0,0 +1,5 @@
+mkdocs-material~=4.4.0
```

Comment:
you could probably rename this file to just `reuiements.txt` given that its in the `mkdocs` directory now :)

Reply:
Good point haha. Thanks!